<launch>

  <arg name="model" default="burger"/>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find self_topoexplore)/rviz/topoexplore.rviz" />


  <node name="map_building" pkg="self_topoexplore" type="robot_no_refine.py" output="screen">
    <param name="network" value="retrievalSfM120k-vgg16-gem" />
    <param name="platform" value="cuda" />
    <param name="robot_name" value="robot1"/>
    <param name="robot_num" value="1"/>
  </node>

  <node name="experiment" pkg="self_topoexplore" type="experiment.py" output="screen">
    <param name="robot_num" value="1"/>
  </node>

 <!-- <include file="$(find self_topoexplore)/launch/includes/move_base.launch">
 </include> -->

</launch>